File: app-components/src/main/java/org/synsystems/onlypass/components/cryptography/encryption/aes/DerivedKeyCredentialConverterModule.java
package org.synsystems.onlypass.components.cryptography.encryption.aes;

import org.synsystems.onlypass.components.AppScope;

import dagger.Module;
import dagger.Provides;

@Module
public class DerivedKeyCredentialConverterModule {
  @Provides
  @AppScope
  public DerivedKeyCredentialConverter provideDerivedKeyCredentialConverter() {
    return new DerivedKeyCredentialConverter();
  }
}

File: app-components/src/main/java/org/synsystems/onlypass/components/cryptography/encryption/aes/DerivedKeyCredentialConverterModule.kt
package org.synsystems.onlypass.components.cryptography.encryption.aes

import dagger.Module
import dagger.Provides
import org.synsystems.onlypass.components.AppScope

@Module
class DerivedKeyCredentialConverterModule {

  @Provides
  @AppScope
  fun provideDerivedKeyCredentialConverter() = DerivedKeyCredentialConverter()
}